aopy/precondition.py

```diff
@@ -162,5 +162,29 @@ def get_psd_welch(data, fs,n_freq = None):
         f, psd = signal.welch(data, fs, average = 'median')
     return f,psd
 
+def bin_spikes(data, fs, bin_width):
+    '''
+    bins spikes
+
+    Args:
+        data (ndarray): time series spike data (nt x 1).
+        fs (float): sampling rate of data
+        bin width (int): width of bin
+
+    Returns:
+        binned_spikes (ndarray): binned spikes 
+    '''
+
+    dT = 1/fs # time width of bin
+    nbins = math.ceil(len(data)/bin_width) # the number of bins
+    binned_spikes = np.zeros(nbins)
```

Comment:
here you can adjust the shape to be `np.zeros((nbins, nch))` where nch is the size of the second dimension `data.shape[1]`